taskweaver/llm/google_genai.py

```diff
@@ -0,0 +1,160 @@
+import json
+from typing import Any, Generator, List, Optional
+
+import google.generativeai as genai
+import requests
```

Comment:
Considering those models are optional ones, maybe we should not have them in required dependency but put the model import into constructor which only checks the availability of such dependency when the user has actually configured to use it (and print an error message instruct user to install corresponding package accordingly)?

Reply:
We may also need a page to list all the evaluated models and their expected behaviors.

Reply:
> Considering those models are optional ones, maybe we should not have them in required dependency but put the model import into constructor which only checks the availability of such dependency when the user has actually configured to use it (and print an error message instruct user to install corresponding package accordingly)?

agree, should apply to all the optional functions, including UI, embedding, LLMApis.